docs/source/overview.rst

```diff
@@ -0,0 +1,305 @@
+
+See :ref:`managed-jobs` and :ref:`many-jobs` to get started.
+
+.. _concept-services:
+
+Services
+--------
+
+A *service* is for AI model serving.
+A service can have one or more replicas, potentially spanning across locations (regions, clouds, clusters), pricing models (on-demand, spot, etc.), or even GPU types.
+
+See :ref:`sky-serve` to get started.
+
+Bringing your infra
+-------------------------------------------------------------------
+
+SkyPilot easily connects to your existing infra---clouds, Kubernetes
+clusters, or on-prem machines---using each infra's native authentication
+(cloud credentials, kubeconfig, SSH).
+
+Cloud VMs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SkyPilot can launch VMs on the clouds and regions you have access to.
+Run ``sky check`` to check access.
+
+SkyPilot supports most major cloud providers. See :ref:`cloud-account-setup` for details.
+
+.. raw:: html
+
+   <p align="center">
+   <picture>
+      <img class="only-light" alt="SkyPilot Supported Clouds" src="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/cloud-logos-light.png" width=85%>
+      <img class="only-dark" alt="SkyPilot Supported Clouds" src="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/cloud-logos-dark.png" width=85%>
+   </picture>
+   </p>
+
+By default, SkyPilot reuses your existing cloud authentication methods.  Optionally, you can also :ref:`set up <cloud-permissions>` specific roles, permissions, or service accounts for SkyPilot to use.
+
+.. _concept-kubernetes-clusters:
+
+Kubernetes clusters
+~~~~~~~~~~~~~~~~~~~~~
+
+You can bring existing Kubernetes clusters, including managed clusters (e.g.,
+EKS, GKE, AKS) or on-prem ones, into SkyPilot.  Auto-failover
+between multiple Kubernetes clusters is also supported.
+
+See :ref:`kubernetes-overview`.
+
+.. figure:: images/k8s-skypilot-architecture-dark.png
+   :width: 50%
+   :align: center
+   :alt: SkyPilot on Kubernetes
+   :class: no-scaled-link, only-dark
+
+   SkyPilot layers on top of your Kubernetes cluster(s).
+
+.. figure:: images/k8s-skypilot-architecture-light.png
+   :width: 50%
+   :align: center
+   :alt: SkyPilot on Kubernetes
+   :class: no-scaled-link, only-light
+
+   SkyPilot layers on top of your Kubernetes cluster(s).
+
+.. _concept-existing-machines:
+
+Existing machines
+~~~~~~~~~~~~~~~~~~~~~
+
+If you have existing machines, i.e., a list of IP addresses you can SSH into, you can bring them into SkyPilot.
+
+See :ref:`Using Existing Machines <existing-machines>`.
+
+.. figure:: images/sky-existing-infra-workflow-light.png
+   :width: 85%
+   :align: center
+   :alt: Deploying SkyPilot on existing machines
+   :class: no-scaled-link, only-light
+
+.. figure:: images/sky-existing-infra-workflow-dark.png
+   :width: 85%
+   :align: center
+   :alt: Deploying SkyPilot on existing machines
+   :class: no-scaled-link, only-dark
+
+SkyPilot's cost and capacity optimization
```

Comment:
This section feels a bit disconnected with the sections above. Not sure if there is a better way to do it.

Reply:
Could maybe keep the section but rename it "Unified Compute Pool" (or "The Sky: optimized, unified compute pool", or similar), so that it maps directly onto the leading figure at the top of the page.
The section does seem pretty valuable to keep in the doc.

Reply:
Tried rewording the first para. Not sure if it's better?

Reply:
Yes, seems better